Source: written by chithanh@gentoo.org, from Gentoo Bug #348552
Upstream: no
Reason: Fix compilation with llvm-2.8 and gcc-4.5
diff -ur a/src/scripting/abc.h b/src/scripting/abc.h
--- a/src/scripting/abc.h	2010-12-08 17:55:46.000000000 +0100
+++ b/src/scripting/abc.h	2010-12-12 21:49:56.549000030 +0100
@@ -19,6 +19,7 @@
 
 #ifndef _ABC_H
 #define _ABC_H
+#define alignof _alignof
 
 #include "compat.h"
 #include <cstddef>
